edenai_apis/apis/groq/groq_api.py

```
@@ -0,0 +1,172 @@
+from edenai_apis.llmengine.llm_engine import LLMEngine
+from edenai_apis.loaders.data_loader import ProviderDataEnum
+from edenai_apis.loaders.loaders import load_provider
+from edenai_apis.utils.types import ResponseType
+from edenai_apis.features.llm.llm_interface import LlmInterface
+from edenai_apis.features.llm.chat.chat_dataclass import ChatDataClass
+
+
+class GroqApi(ProviderInterface, TextInterface, LlmInterface):
+    provider_name = "groq"
+
+    def __init__(self, api_keys: Dict = {}) -> None:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)